src/atomic/pag.login/login.component.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { useTheme } from 'styled-components';
+
+import { AtomSeparator } from '../atm.separator/separator.component';
+import { AtomTitle } from '../atm.title/title.component';
+import { MoleculeCallToActionButton } from '../mol.button/button.component';
+import { MoleculeNamedInput } from '../mol.named-input/named-input.component';
+
+import { StylePageLoginContainer } from './login.component.style';
+
+export const PageLogin: React.FC = () => {
+  const { spacing, colors } = useTheme();
+  return (
+    <StylePageLoginContainer>
+      <AtomSeparator size={spacing['2-xl']} />
+      <AtomTitle>Bem vindo(a) à Taqtitle!</AtomTitle>
+      <AtomSeparator size={spacing['2-xl']} />
+      <MoleculeNamedInput>Email</MoleculeNamedInput>
+      <AtomSeparator size={spacing['2-xl']} />
+      <MoleculeNamedInput>Password</MoleculeNamedInput>
+      <AtomSeparator size={spacing['2-xl']} />
+      <MoleculeCallToActionButton title="Entrar" color={colors.purple[400]} onPress={() => {}} />
```

Comment:
Será que faz sentido ter botões call to action de diferentes cores?

Reply:
A intenção era criar um botão que pode ter várias cores, porém eu esqueci de trocar o nome. O que vc acha disso? Eu crio um botão pra cada(primary, secondary, call to action) ou eu crio um botão que pode ter cor primary, secondary e call to action?

Reply:
Pra projeto de onboard eu diria que tanto faz, só tinha achado estrnaho o botão ser call-to-action e ter cores diferentes mesmo. Nos projetos usamos a primeira abordagem.